src/libUtils/BlockTransactionsHelper.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2022 Zilliqa
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#ifndef ZILLIQA_SRC_LIBUTILS_BLOCKTRANSACTIONSHELPER_H_
+#define ZILLIQA_SRC_LIBUTILS_BLOCKTRANSACTIONSHELPER_H_
+
+#include <boost/optional.hpp>
+#include <string>
+
+class TxBlockChain;
+
+class BlockTransactionsHelper {
+ public:
+  static boost::optional<uint32_t> GetTransactionIndexInBlock(
```

Comment:
nit: you don't need a separate class for a utility function. Especially when it is used in one .cpp file - can keep it there as 'static' or in an anonymous namespace.